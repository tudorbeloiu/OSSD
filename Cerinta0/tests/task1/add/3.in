1
1
5
11
9000
12
8200
13
8193
14
8190
15
9
