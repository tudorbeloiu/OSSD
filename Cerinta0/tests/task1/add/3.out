11: (0, 0)
12: (0, 0)
13: (0, 0)
14: (0, 1023)
15: (0, 0)
